### ISSUE TITLE
Org Profile page does NOT refresh, as the post of the selected ORG are not displayed.

### ISSUE THREAD

**Issue by rabbi08** [FightPandemics/FightPandemics#1545]
1. Open "Staging.fightpandemics.com"
2. Log in using valid username/pwd
3.  Navigate to Help board-> click any random user profile to view -> Navigate to profile and click on any own Org to view the Org profile, following issues observed:

1. The navigation side bar doesn’t close after selection as it do, when user click on "Profile" to view its own profile.
2. So, now we have to clicks on ‘x’ to close navigation bar, you will observe that name of the users own org is displayed but the post displayed are from previous visited user profile. Hence the profile page for Org is not updated.

Refer below gif:
https://share.getcloudapp.com/eDuPW54K

Screenshot:
https://share.getcloudapp.com/nOun6RZn

Note: This issue exist on both desktop and mobile. Desktop screenshot
https://share.getcloudapp.com/xQuAPwDx

This issue might have been introduced due to pagination, #927, related PR #1274.

**Comment by rabbi08** [comment]
PR #1274 also introduce below issue:

Tested this PR on mobile and Desktop in STG:

- Desktop version is working fine: PASS
- Clicking on company profile on mobile, the UI is not distorted as reported above: PASS
- But on the mobile the loading spinner exist along with the message about no post. "Filter" is not working in STG on mobile. : FAIL
Refer below screenshot:
Devices Tested:
iphone 11 pro max
Tested on simulator: Samsung Galaxy, iphone XR
Android:
https://share.getcloudapp.com/NQu1LrR6
iOS:
https://share.getcloudapp.com/OAuJ0qxx

**Comment by joshmorel** [comment]
@OzkanAbdullahoglu also as part of this fix. For profile activity also if there is no activity just leave it blank like current state:

![image](https://user-images.githubusercontent.com/10546720/94139869-5bcce600-fe38-11ea-8f3c-8c788237e12e.png)

Instead of :
![image](https://user-images.githubusercontent.com/10546720/94139905-6c7d5c00-fe38-11ea-856d-089112ced83a.png)

CC @sheeba1

**Comment by OzkanAbdullahoglu** [comment]
@joshmorel  and what about Org Profile activity? Should I keep it blank too?

**Comment by OzkanAbdullahoglu** [comment]
> 1. Open "Staging.fightpandemics.com"
> 2. Log in using valid username/pwd
> 3. Navigate to Help board-> click any random user profile to view -> Navigate to profile and click on any own Org to view the Org profile, following issues observed:
> 4. The navigation side bar doesn’t close after selection as it do, when user click on "Profile" to view its own profile.
> 5. So, now we have to clicks on ‘x’ to close navigation bar, you will observe that name of the users own org is displayed but the post displayed are from previous visited user profile. Hence the profile page for Org is not updated.
> 
> Refer below gif:
> https://share.getcloudapp.com/eDuPW54K
> 
> Screenshot:
> https://share.getcloudapp.com/nOun6RZn
> 
> Note: This issue exist on both desktop and mobile. Desktop screenshot
> https://share.getcloudapp.com/xQuAPwDx
> 
> This issue might have been introduced due to pagination, #927, related PR #1274.

I  think I covered all of them, BTW article 5 is not about pagination. It is still there before #1274 
Pls check and advise If there are any more bug.

**Comment by rabbi08** [comment]
@OzkanAbdullahoglu @joshmorel 

Verified below issues on the feature build:
1. The Org profile page refresh when Org is selected from the dropdown after viewing other profiles.: PASS
Refer below GIF
https://share.getcloudapp.com/jkuDdeO7

2. Filter Post -> Loading spinner on Mobile is working fine: PASS
Refer below GIF
https://share.getcloudapp.com/geuqv42q

3. The activity page is Blank for the individual user when there is no post to display: PASS
Refer below screenshot.
https://share.getcloudapp.com/7Ku8epEP

4. The "No post and create post" message is displayed for Org Profile as opposite behavior to the Individual Profile : FAIL
I would say we should leave the activity area for Org as BLANK just like for Individual profile.
Refer below screenshot.
https://share.getcloudapp.com/4guJQ12b

and Article#5 issue exist because of #1, now that is fix so #5 issue is not observed.

**Comment by OzkanAbdullahoglu** [comment]
> 4. The "No post and create post" message is displayed for Org Profile as opposite behavior to the Individual Profile : FAIL
>    I would say we should leave the activity area for Org as BLANK just like for Individual profile.
>    Refer below screenshot.
>    https://share.getcloudapp.com/4guJQ12b
> 
> and Article#5 issue exist because of #1, now that is fix so #5 issue is not observed.

@rabbi08 Thank you for your review. I will handle #4 and commit. 
#5 existed because of below code, there was no trigger in NavItemBrief (sub-menu of organisation)

  ```
<NavItemBrief
                history={history}
                key={organisation._id}
                onClick={toggleDrawer} //This line
              >
```
Maybe am I missing something more important here?
